src/sentry/static/sentry/app/components/avatar/userAvatar.jsx

```diff
@@ -53,6 +53,8 @@ class UserAvatar extends React.Component {
         tooltip={
           typeof renderTooltip === 'function'
             ? renderTooltip(user)
+            : props.tooltip
```

Comment:
@markstory @billyvg The `tooltip` prop wasn't properly forwarded from here https://github.com/getsentry/sentry/blob/f365c0b8f356fc131b45dc091530674785701a56/src/sentry/static/sentry/app/views/organizationReleases/detail/commitAuthorStats.jsx#L124

So, I'm fixing that here in case this component was consumed with a `tooltip` prop.